src/Analytics/Schema/ContextModule.ts

```diff
@@ -0,0 +1,8 @@
+export interface ContextModule {
```

Comment:
@alloy - interesting, so the idea is to think of `/Schema` as a collection of interfaces?

Reply:
Yep! And what I ended up doing is defining the possible schema defs as a union (Trackables), meaning that as you add properties to the object TS narrows it down to a single schema def in the union (type narrowing).